packages/@sfx/search-data-source-plugin/src/search-data-source-plugin.ts

```diff
@@ -0,0 +1,45 @@
+import { Plugin, PluginRegistry, PluginMetadata } from '@sfx/core';
+import { BridgeConfig, BrowserBridge, Query } from 'groupby-api';
+
+/**
+ * This plugin is responsible for exposing an instance of search browserBridge
```

Comment:
```suggestion
 * This plugin exposes an instance of the GroupBy search client
```

Also fix the wrapping if necessary.